src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs

```diff
@@ -181,13 +181,13 @@ partial class ApplicationEngine
             [OpCode.SIZE] = 150,
             [OpCode.HASKEY] = 270000,
             [OpCode.KEYS] = 500,
-            [OpCode.VALUES] = 7000,
+            [OpCode.VALUES] = 270000,
             [OpCode.PICKITEM] = 270000,
-            [OpCode.APPEND] = 15000,
+            [OpCode.APPEND] = 270000,
             [OpCode.SETITEM] = 270000,
-            [OpCode.REVERSEITEMS] = 500,
+            [OpCode.REVERSEITEMS] = 270000,
             [OpCode.REMOVE] = 500,
-            [OpCode.CLEARITEMS] = 400,
+            [OpCode.CLEARITEMS] = 270000,
```

Comment:
Why it cost so much?

Reply:
In the worst case, clearing a 2048-sized map full of structs will be very time consuming (averagely 300 μs per instruction on my computer)

Reply:
https://github.com/neo-project/neo-vm/blob/a14b934f5b5c8779164e3b96b276984b3a57ccd4/src/neo-vm/ExecutionEngine.cs#L1188-L1193

Reply:
Map.Clear can be time consuming:
https://github.com/neo-project/neo-vm/blob/a14b934f5b5c8779164e3b96b276984b3a57ccd4/src/neo-vm/Types/Map.cs#L85

Reply:
Maybe we can reset `CLEARITEMS`  fee. On the one hand, it's low frequency operation, and on the other hand, then if there is repeated operation, the second time is empty.

Reply:
Have reverted